src/Cache/Server/Server.php

```diff
@@ -0,0 +1,576 @@
+<?php # -*- coding: utf-8 -*-
+
+namespace Inpsyde\MultilingualPress\Cache\Server;
+
+use Inpsyde\MultilingualPress\Cache\CacheFactory;
+use Inpsyde\MultilingualPress\Cache\Driver\CacheDriver;
+use Inpsyde\MultilingualPress\Cache\Pool\CachePool;
+
+/**
+ * @package Inpsyde\MultilingualPress\Cache
+ * @since   3.0.0
+ */
+class Server {
+
+	const UPDATING_KEYS_TRANSIENT = 'mlp_cache_server_updating_keys';
+	const SPAWNING_KEYS_TRANSIENT = 'mlp_cache_server_spawning_keys_';
+	const HEADER_KEY = 'X-MLP-Cache-Update-Key';
```

Comment:
Let's use `MLP-*` header names, and no `X-*` ones, see [here](https://tools.ietf.org/html/rfc6648).